Simcoe/Simcoe.swift

```diff
@@ -134,8 +138,10 @@ public final class Simcoe {
     /// - Parameters:
     ///   - product: The SimcoeProductConvertible instance.
     ///   - eventProperties: The event properties.
-    func logRemoveFromCart<T: SimcoeProductConvertible>(_ product: T, eventProperties: Properties?) {
-        let providers: [CartLogging] = findProviders()
+    ///   - providers: The providers.
+    func logRemoveFromCart<T: SimcoeProductConvertible>(_ product: T,
+                           eventProperties: Properties?, providers: [AnalyticsTracking]? = nil) {
+        let providers: [CartLogging] = find(providers)
         let simcoeProduct = product.toSimcoeProduct()
         let propertiesString = eventProperties != nil ? "=> \(eventProperties!.description)" : ""
```

Comment:
Because of the differences in formatting (based on whether eventProperties exists), we would not be able to use that logic in this case